### DISCUSSION TITLE
what language should we use to build?

### DISCUSSION THREAD

**Discussion by RishithaRao1** [microsoft/AI-Chat-App-Hack#17] [Q&A] [answered]
do we have to build the app only in python?

**Comment by pamelafox** [comment, marked as answer]
We will be focusing on Python in our live streams, but you're welcome to build a RAG chat app in other languages. You can find examples for .NET, Java, and JavaScript here:
https://learn.microsoft.com/en-us/azure/developer/intro/azure-ai-for-developers
We will also have a talk by the JS developers, about the web components they developed.
